src/backend/server.ts

```diff
@@ -222,7 +220,7 @@ const server = (async () => {
     const request = new Request(url.href, {
       method: 'POST',
       headers: encoder.headers,
-      body: Readable.from(encoder.encode()),
+      body: JSON.stringify(encoder.encode()),
```

Comment:
I'm actually not sure about this change. Could you @LarsTheGlidingSquirrel test it locally to see if it works?